activedata_etl/transforms/jscov_to_es.py

```diff
@@ -32,42 +35,60 @@ def count_generator():
             count += 1
 
     def generator():
-        response_stream = http.get(artifact.url).raw
-        with Timer("Processing {{jscov_file}}", param={"jscov_file": artifact.url}):
-            counter = count_generator().next
-
-            for source_file_index, obj in enumerate(stream.parse(response_stream, [], ["."])):
-                if please_stop:
-                    Log.error("Shutdown detected. Stopping job ETL.")
-
-                if source_file_index == 0:
-                    # this is not a jscov object but an object containing the version metadata
-                    # TODO: this metadata should not be here
-                    # TODO: this version info is not used right now. Make use of it later.
-                    jscov_format_version = obj.get("version")
-                    continue
-
-                try:
-                    for d in process_source_file(
-                        artifact_etl,
-                        counter,
-                        obj,
-                        task_cluster_record
-                    ):
-                        yield d
-                except Exception as e:
-                    Log.warning(
-                        "Error processing test {{test_url}} and source file {{source}} while processing {{key}}",
-                        key=source_key,
-                        test_url=obj.get("testUrl"),
-                        source=obj.get("sourceFile"),
-                        cause=e
-                    )
+        with ZipFile(jsdcov_file) as zipped:
+            for num, zip_name in enumerate(zipped.namelist()):
+                if num == 1:
```

Comment:
this check is from the other code, which was to ensure there is only one file in the zipfile.  There are multiple files in this zipfile, so remove this check.